src/test/java/org/zalando/planb/revocation/domain/RevokedClaimsInfoTest.java

```diff
@@ -33,8 +22,8 @@
     // Workaround to add double quotes around claim names (for JSON comparison)
     static {
         StringBuilder jsonNamesBuilder = new StringBuilder();
-        for (int i = 0; i < NAMES.length; i++) {
-            jsonNamesBuilder.append("\"" + NAMES[i] + "\",");
+        for (String NAME : NAMES) {
```

Comment:
I'm pretty sure, because Arrays have an order and the enhanced for loop should keep it. I made these changes, because IntelliJ marked the lines yellow ;-) They have nothing todo with the actual problem